src/Dfe.Spi.UkrlpAdapter.Infrastructure.UkrlpSoapApi/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Dfe.Spi.UkrlpAdapter.Infrastructure.UkrlpSoapApi.UnitTests")]
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
A new one for me - wasn't even even aware that this was possible!

Of course, it allows you to unit test internal stuff - I guess this is why I generally make stuff public if I want to test it, but this works just as well. Interesting.

Reply:
Yeah. Slightly moot in cases like this, as public to just your own stuff makes little difference. Just makes me feel better mostly